public/app/features/alerting/unified/components/export/providers.ts

```diff
@@ -32,12 +43,51 @@ export const HclExportProvider: ExportProvider<'hcl'> = {
   type: 'terraform',
 };
 
+export const OperatorExportProvider: ExportProvider<'operator'> = {
+  name: 'Grafana Operator',
+  exportFormat: 'operator',
+  overrideLanguage: 'yaml',
+  type: 'operator',
+  supports: ['AlertRuleGroup', 'AlertRule', 'Receiver', 'RuleFolder'],
+  formatter: (raw: string) => {
+    const out: any[] = [];
```

Comment:
Drone does not like the any
```suggestion
    const out: Record<string, unknown>[] = [];
```